RemixedDungeon/src/main/java/com/watabou/pixeldungeon/actors/buffs/NatureParalysis.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Pixel Dungeon
+ * Copyright (C) 2012-2014  Oleg Dolya
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>
+ */
+package com.watabou.pixeldungeon.actors.buffs;
+
+import com.nyrds.pixeldungeon.ml.R;
+import com.watabou.noosa.Game;
+import com.watabou.pixeldungeon.actors.Char;
+import com.watabou.pixeldungeon.items.rings.RingOfElements.Resistance;
+import com.watabou.pixeldungeon.sprites.CharSprite;
+import com.watabou.pixeldungeon.ui.BuffIndicator;
+
+public class NatureParalysis extends FlavourBuff {
+
+	private static final float DURATION	= 10f;
+	
+	@Override
+	public boolean attachTo( Char target ) {
+		if (super.attachTo( target )) {
+			target.paralyse(true);
+			return true;
+		} else {
+			return false;
+		}
+	}
+	
+	@Override
+	public void detach() {
+		target.paralyse(false);
+		super.detach();
+	}
+	
+	@Override
+	public int icon() {
+		return BuffIndicator.PARALYSIS;
```

Comment:
Same buff icon for two different buffs would be confusing

Reply:
Aight! I'll fix all of these, and add a few other tweaks. (Should be fixed and ready by tomorrow)

Reply:
Sweet

Reply:
Alright, I reverted GasParalysis back into just Paralysis;

Removed the new potion and immunity for now, since potion of purity already exists.